buildSrc/src/main/groovy/nvadatamodel.java-conventions.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
 }
 
 group 'com.github.bibsysdev'
-version '0.14.23'
+version '0.14.24'
```

Comment:
Probably this is 0.15.0, since the new version is not compatible with the old one